sklearn/manifold/_isomap.py

```diff
@@ -123,11 +130,13 @@ class Isomap(TransformerMixin, BaseEstimator):
     .. [1] Tenenbaum, J.B.; De Silva, V.; & Langford, J.C. A global geometric
            framework for nonlinear dimensionality reduction. Science 290 (5500)
     """
+    # Once positional arguments are no longer supported, we should move `radius`
```

Comment:
I apologize for asking you to add that, this is already obsolete.
Positional arguments were planned to raise an error in the next version. This decorator was actually removed in #20002, so you can safely:
- rebase your branch on the main branch
- remove this comment
- reorder the parameters in the most intuitive way (matching the order in __init__ and in the docstring)